changelogs/unreleased/8736-implement-graphql-endpoint.yml

```diff
@@ -0,0 +1,7 @@
+description: Implemented a GraphQL endpoint.
+  Added a query to fetch environments that supports filtering, sorting and paging.
+issue-nr: 8736
+change-type: minor
+destination-branches: [master]
+sections:
+  feature: "{{description}}"
```

Comment:
I'm not sure how relevant this is to end users. If I'm not mistaken the FE team will be the only consumer of this endpoint ?

Reply:
I'm not sure if we want to announce this feature to the end users as well. Wdyt @wouterdb ?

Reply:
I think it can be added as a minor feature or feature, end-user may want to know